demo/opae-nlb-demo/Dockerfile

```diff
@@ -1,24 +1,10 @@
-# CLEAR_LINUX_BASE and CLEAR_LINUX_VERSION can be used to make the build
-# reproducible by choosing an image by its hash and installing an OS version
-# with --version=:
-# CLEAR_LINUX_BASE=clearlinux@sha256:b8e5d3b2576eb6d868f8d52e401f678c873264d349e469637f98ee2adf7b33d4
-# CLEAR_LINUX_VERSION="--version=29970"
-#
-# This is used on release branches before tagging a stable version.
-# The main branch defaults to using the latest Clear Linux.
-ARG CLEAR_LINUX_BASE=clearlinux:latest
+FROM fedora:32 as builder
```

Comment:
Perhaps a comment here would be useful about the choice of fedora. Otherwise future me may wonder why debian is not used for the builder container too.

Reply:
perhaps also:
```
ARG FEDORA_RELEASE="34"

FROM fedora:${FEDORA_RELEASE} as builder
```

Reply:
thank you for the review! done.